test/v2/client/test_client.go

```diff
@@ -518,67 +558,138 @@ func (c *TestClient) ReadBlobFromRelays(
 	key corev2.BlobKey,
 	relayKeys []corev2.RelayKey,
 	expectedPayload []byte,
-	blobLengthSymbols uint32) error {
+	blobLengthSymbols uint32,
+	timeout time.Duration) error {
```

Comment:
consider documenting that this is the timeout for each individual relay call, as opposed to a timeout for the entire `ReadBlobFromRelays`, which is what I would assume by default

Reply:
```
// The timeout provided is a timeout for each individual relay read, not all reads as a whole.
```